src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -143,6 +142,10 @@ public override void Load()
                 // Instantly re-throw the exception
                 throw;
             }
+            catch (FallbackClientLookupException ex)

```

Comment:
What ? How is okay to continue? Why do we even let this exception propagate this high? The only purpose is for logging? If so, we can log in manager.

Reply:
You are right, I was thoughtless, doing this is not reasonable.